src/nodeTypesStore.js

```diff
@@ -31,6 +31,7 @@ export default new Vuex.Store({
           label: nodeType.label,
           bpmnType: nodeType.bpmnType,
           rank: nodeType.rank || BOTTOM,
+          items: nodeType.items?.map(item => ({ ...item, type: item.id })),
```

Comment:
this `?` will not work with the current version of babel, it will cause some issues when running the bundle as seen in SB. Can you please re-check if the `npm run build-bundle` work as expected?